azkaban-common/src/main/java/azkaban/executor/ExecutableFlow.java

```diff
@@ -211,6 +213,10 @@ public void setAzkabanFlowVersion(final double azkabanFlowVersion) {
     this.azkabanFlowVersion = azkabanFlowVersion;
   }
 
+  public boolean isLocked() { return isLocked; }
```

Comment:
Please follow the style guide in `CONTRIBUTING.md` to add `this` for field access. You can also enable` save action `to automatically do so.